packages/js/src/ui/components/InboxTabs/MoreTabsDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import { Accessor, Setter } from 'solid-js';
+import { cn, useStyle } from 'src/ui/helpers';
+import { Check, DotsMenu } from '../../icons';
+import { Dropdown, dropdownItemVariants, Tabs } from '../primitives';
+
+export const MoreTabsDropdown = (props: {
+  dropdownTabs: Accessor<Array<{ label: string; value: Array<string> }>>;
+  activeTab: Accessor<string>;
+  setActiveTab: Setter<string>;
+}) => {
+  const style = useStyle();
+
+  return (
+    <Dropdown.Root fallbackPlacements={['bottom', 'top']} placement="bottom-start">
+      <Dropdown.Trigger
+        class={style('actions__dropdownTrigger')}
```

Comment:
I've renamed the `moreActions` to `actions` just to be able to reuse the same appearance keys. 
IDK if it's ok, ideally we should have a flexible component that we can reuse in all places, but that would require more refactoring.

Reply:
⚠️ This is not ok, `moreActions__dropdownTrigger` is a more specific descriptor reserved for that dropdown trigger. You should use something like `moreTabs__dropdownTrigger` here. You can target both with `dropdownTrigger`, as that will be applied to both.

Also, you should use the `appearanceKey` prop here as this `class` attribute will be overriden by the class of the button that you specified in `asChild`. Not 100% sure, but make sure to double check that the correct appearance keys and styles are applied.

Reply:
@desiprisg Visually these two dropdowns look the same that's why I thought about using the same appearance key. We may need to extract it to a separate component and reuse it in both places, then it will have a better sense.

Reply:
Appearance keys are not really about things looking the same, but more about allowing developers to target them specifically. It does not need to be a different component for us. `actions` does not describe the element here. `moreActions` was about the extra actions you can access via the first dropdown that we added. This is about tabs.

Reply:
But you want to target the same components (visually same) with one appearance prop. I want to adjust all my "..." dropdowns styles to look the same.

Reply:
Yeah you can do that, you always separate those with `__` and pass the most specific to the element. Just make sure the names make sense.